### ISSUE TITLE
Resource bundle not found in application

### ISSUE THREAD

**Issue by lepetitroro** [mapbox/DEPRECATED-mapbox-ios-sdk#396]
Hello, 

I updated mapbox in the submodule in my project, the compilation has succeeded, but when i run my app this error occurs: "**\* Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Resource bundle not found in application.". 

I followed the installation procedure but it doesn't work. 
I work on the develop branch. everything is updated. 

thank you for your help!
Romain

**Comment by incanus** [comment]
You're getting that error because you don't have the resource bundle installed right. See here: 

https://www.mapbox.com/mapbox-ios-sdk/#source

> Add the Mapbox.bundle product from the SDK to the Copy Bundle Resources build phase of your app target. This way, the bundle will be copied at build time out of the SDK build products folder and included as a resource in your app product.

**Comment by lepetitroro** [comment]
Hi,

I have already done it. I cleaned my build folder and now i have another problem : libMapbox.a, file was built for archive which is not the architecture being linked (i386) ...

Thanks for your help ...

Romain.

**Comment by incanus** [comment]
@lepetitroro please send a support issue in to support@mapbox.com about this and we'll try to help. The GitHub issue queue is more for discussing development-related issues and not general usage problems.

**Comment by heatherSnepenger** [comment]
Was there any resolution to this issue? I'm experiencing the same problem.

**Comment by incanus** [comment]
It's not an outstanding issue — it is configuration of your project and not including `Mapbox.bundle` properly. If you have an issue @heatherSnepenger, please feel free to write to support and we can address the specifics in your project.

**Comment by beweinreich** [comment]
@incanus It looks like the issue @heatherSnepenger and others are having is related to this comment https://github.com/mapbox/mapbox-ios-sdk/issues/580#issuecomment-89436801

I changed
`[[NSBundle mainBundle] pathForResource:@"Mapbox" ofType:@"bundle"]`

to 
`[[NSBundle bundleForClass:[self class]] pathForResource:@"Mapbox" ofType:@"bundle"]`

in RMMapView.m and the bug was fixed.

Edit: I see that you guys are actively discussing it over on that thread. I'll check it out over there..

**Comment by MikeInSwitzerland** [comment]
I had this error when I made a complete beginner's mistake..  I had saved the SDK in a folder with a space in it  ("mapbox-ios-sdk (raw)").   My app built perfectly, it started running, then threw an exception, saying "Mapbox.bundle" couldn't be found.  I renamed the folder to "mapbox-ios-sdk-raw", re-added it to my app, and it all worked fine.

Remind me again, what year is this...?
(Sigh.)
